code/game/turfs/closed/minerals.dm

```diff
@@ -36,7 +36,7 @@
 	M.Translate(-4, -4)
 	transform = M
 	icon = smooth_icon
-
+	AddElement(/datum/element/tool_bump, tool_behaviours = list(TOOL_MINING), require_active_hand = FALSE)
```

Comment:
Bespoke elements work based off checking if any existing elements of that type share the same set of arguments.
Unfortunately, they don't care about the contents of a list, they only care about the EXACT same list
So for elements like this we pass in locally static lists, to ensure the memory benefits are actually felt 
```suggestion
	var/static/list/behaviors = list(TOOL_MINING)
	AddElement(/datum/element/tool_bump, tool_behaviours = behaviors, require_active_hand = FALSE)
```